dlt/destinations/impl/databricks/sql_client.py

```diff
@@ -65,6 +66,18 @@ def rollback_transaction(self) -> None:
     def native_connection(self) -> "DatabricksSqlConnection":
         return self._conn
 
+    def has_dataset(self) -> bool:
+        query = """
+    SELECT CATALOG_NAME
+    FROM INFORMATION_SCHEMA.SCHEMATA
+    WHERE """
+        db_params = self.fully_qualified_dataset_name(escape=False).split(".", 2)
+        if len(db_params) == 2:
+            query += " catalog_name = %s AND "
+        query += "schema_name = %s"
```

Comment:
Since databricks always uses `catalog_name` I would skip this `if` and just hardcode the whole query, it's easier to read.